rules/sonic-fips.mk

```diff
@@ -3,7 +3,7 @@
 ifeq ($(BLDENV), bookworm)
 FIPS_VERSION = 1.5.2
 FIPS_OPENSSL_VERSION = 3.0.11-1~deb12u2+fips
-FIPS_OPENSSH_VERSION = 9.2p1-2+deb12u3+fips
+FIPS_OPENSSH_VERSION = 9.2p1-2+deb12u5+fips
```

Comment:
We can't make this change, the 9.2p1-2+deb12u5+fips package does not exist
If we want do this upgrade, we need upgrade package in sonic-fips repo first, then upload the package first.

Reply:
Here is sonic-fips repo make file: https://github.com/liuh-80/sonic-fips/blob/main/rules/openssh.mk
Latest version still 9.2p1-2+deb12u3

Also, the openssh submodule still point to deb12u3 version.